index.html

```diff
@@ -8216,6 +8216,95 @@ <h5>Presentational Roles Conflict Resolution</h5>
 				</tbody>
 			</table>
 		</div>
+		<div class="role" id="suggestion">
+			<rdef>suggestion</rdef>
+			<div class="role-description">
+					<p>A suggestion contains a proposed change to content that can be approved or rejected.
```

Comment:
In the explainer, it states:

> role="suggestion" -- used to group proposed changes in a document (role "deletion" and "insertion" children).

So I was envisioning (potentially) multiple "deletion" and/or "insertion" children. But from the wording above, my impression is that a given suggestion is limited to a single change. Which is it? And can we make the spec language more explicit so that there is no question?

Reply:
It's for a single change, so the possibilities are:
- 1 deletion child
- 1 insertion child
- both a deletion and insertion child

Reply:
I changed the first sentence of the definition to use "single proposed change"